plugins/scaffolder-backend/src/scaffolder/actions/builtin/publish/log.ts

```diff
@@ -0,0 +1,72 @@
+/*
+ * Copyright 2021 Spotify AB
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import { readdir, stat } from 'fs-extra';
+import { relative, resolve } from 'path';
+import { createTemplateAction } from '../../createTemplateAction';
+
+/**
+ * This task is useful for local development and testing of both the scaffolder
+ * and scaffolder templates.
+ */
+export function createPublishLogAction() {
```

Comment:
Very nice addition ⭐ 
Would it be possible to add some tests for it?

Reply:
Fore sure, PR is updated!